docs/PORTING.md

```diff
@@ -1,10 +1,10 @@
-# uVisor Porting Guide
+# ARM uVisor Porting Guide for mbed OS
 
 The uVisor is a low-level security module that creates sandboxed environments on ARM Cortex-M3 and ARM Cortex-M4 devices. Each sandbox can get hold of a set of private resources for which uVisor grants exclusive access. Calls into sandboxed APIs are only possible through uVisor-managed entry points.
 
 The uVisor is not compiled and linked directly into your application. Instead, a glue-layer module compiles the uVisor in the form of a static library, which is periodically released.. This library contains the implementation of all the uVisor APIs and the uVisor core, which is a pre-linked binary component.
```

Comment:
Creates the impression that uvisor is closed source
add sentence after that "you can get the source code here for porting to other polatforms or for compiling it yourself"
@AlessandroA